Mathlib/Algebra/Order/Nonneg/Ring.lean

```diff
@@ -78,12 +78,24 @@ instance orderedCommSemiring [OrderedCommSemiring α] : OrderedCommSemiring { x
     (fun _ _ => rfl) (fun _ _ => rfl)
     (fun _ _ => rfl) (fun _ _ => rfl) fun _ => rfl
 
+instance orderedCommMonoid [OrderedCommSemiring α] : OrderedCommMonoid { x : α // 0 ≤ x } where
+  mul_le_mul_left a _ h c := mul_le_mul le_rfl h a.prop c.prop
```

Comment:
Why do you need this? We already have `Nonneg.orderedCommSemiring`

Reply:
How does that help? There is no `OrderedCommSemiring.toOrderedCommMonoid`. It's only true for the `Nonneg` cone.

Reply:
Oh right!